### PR TITLE
VetTec: When a user updates their bank account information with a blank field, the pre-filled info is still saved #17156

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17156
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17173

## Testing done
- local testing

## Screenshots
screen shot 2019-03-01 at 10 41 54 am
screen shot 2019-03-01 at 10 41 59 am

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/components/PaymentReviewView.jsx [hunk 1]
```diff
@@ -16,16 +16,20 @@ export class PaymentReviewView extends React.Component {
...
-        .toUpperCase()}${name.substring(1)}`;
-      const prefillBankAccount = _.get(
-        this.props.data,
-        'prefillBankAccount',
-        {},
-      );
-      value = _.get(prefillBankAccount, propName, '');
+      const viewBankAccount = _.get(this.props.data, 'view:bankAccount', {});
+
+      if (!hasNewBankInformation(_.get(viewBankAccount, 'bankAccount', {}))) {
```
Comment (rianfowler) [inline src/applications/edu-benefits/0994/components/PaymentReviewView.jsx]: maybe consolidate this into one `_.get(this.props.data, 'view:bankAccount.bankAccount', {})`

**Review by rianfowler — APPROVED** [review]
LGTM. one minor suggestion

**Pull request merged**